From add9717e93f17a19e88bf7950084a7b8c93bd04d Mon Sep 17 00:00:00 2001
From: Kevin Jeon <kevinjeon@google.com>
Date: Fri, 17 Mar 2023 17:47:22 +0000
Subject: [PATCH] DO NOT MERGE: Grant MANAGE_USERS access to Traceur

This change updates the privapp allowlist to grant the MANAGE_USERS
permission to Traceur. This permission is needed to query admin user
status, as Traceur shouldn't be able to start if the current user is not
an admin.

Test: Using ABTD, apply this change with ag/22140600 to verify that
      Traceur still works as intended (opening app, tracing, etc.).
Bug: 262243665
Bug: 262244249
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:1009c5341112e062efa8703516467a7cde72ffa7)
Merged-In: I481c95fed4a364239f0671ed1b4d9bfaa0b07bd6
Change-Id: I481c95fed4a364239f0671ed1b4d9bfaa0b07bd6
---
 data/etc/privapp-permissions-platform.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 813b7995fe89..3dd16721e529 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -516,6 +516,8 @@ applications that come with the platform
         <permission name="android.permission.STATUS_BAR"/>
         <!-- Permissions required to query Betterbug -->
         <permission name="android.permission.QUERY_ALL_PACKAGES"/>
+        <!-- Permissions required to query admin user status -->
+        <permission name="android.permission.MANAGE_USERS"/>
     </privapp-permissions>
 
     <privapp-permissions package="com.android.tv">
-- 
2.41.0.rc0.172.g3f132b7071-goog

